<launch>

  <arg name="input/objects" default="labeled_clusters"/>
  <arg name="output/objects" default="shape_estimeted_objects"/>
  <arg name="use_map_corrent" default="true"/>
  <arg name="map_corrector/use_rad_filter" default="true"/>
  <arg name="map_corrector/rad_threshold" default="1.2217304763960306"/>
  <!-- 70 deg -->
  <node pkg="shape_estimation" type="shape_estimation" name="shape_estimation" output="screen">
    <remap from="input" to="$(arg input/objects)" />
    <remap from="objects" to="$(arg output/objects)" />
    <param name="use_map_corrent" value="$(arg use_map_corrent)" />
    <param name="map_corrector/use_rad_filter" value="$(arg map_corrector/use_rad_filter)" />
    <param name="map_corrector/rad_threshold" value="$(arg map_corrector/rad_threshold)" />
  </node>
</launch>
